### PR TITLE
Add trace events to CASE and PASE Sessions

### PR DESCRIPTION
**Pull request by yyzhong-g**
#### Change overview
Add trace events to CASESession and PASESession for message transfer time analysis.

#### Testing
build on esp32 M5Stack and retrieve trace data after commissioning.

### CODE REVIEW

File: src/protocols/secure_channel/CASESession.cpp [hunk 1]
```diff
@@ -212,6 +213,7 @@ CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddres
                                          uint16_t localSessionId, ExchangeContext * exchangeCtxt,
                                          SessionEstablishmentDelegate * delegate, Optional<ReliableMessageProtocolConfig> mrpConfig)
 {
+    PW_TRACE_INSTANT("CASESession::EstablishSession", "Commissioning");
```
Comment (mspang) [inline src/protocols/secure_channel/CASESession.cpp:216]: "Commissioning" doesn't sound like the right category. CASE is for operational session establishment.

File: src/protocols/secure_channel/CASESession.cpp [hunk 2]
```diff
@@ -235,7 +237,9 @@ CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddres
     SetPeerAddress(peerAddress);
     SetPeerNodeId(peerNodeId);
 
+    PW_TRACE_START("CASESession::SendSigma1", "Commissioning");
```
Comment (mspang) [inline src/protocols/secure_channel/CASESession.cpp:240]: Why is this INSTANT not START/END ?

Why not PW_TRACE_SCOPE inside of SendSigmaR1?

**Review by mspang — COMMENTED** [review]
Please use the appropriate macros.

**Review by mspang — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Review by emargolis — APPROVED** [review]

**Comment by woody-apple** [comment]
@yyzhong-g Can you please restyle this?

**Comment by andy31415** [comment]
fast track: sufficient  time for this to have had cross time review, has several checkmarks, pr is not expected  to affect functionallity except for debug.

**Pull request merged**